omero/changelog.rst

```diff
@@ -0,0 +1,21 @@
+Links to decoupled repositories
+===============================
+
+Since OMERO 5.5, the following repositories have been decoupled.
+
+omero-build:
+------------
+
+- omero-java-gateway :java_gateway_source:`CHANGELOG.md<CHANGELOG.md>`
```

Comment:
`CHANGELOG.md<CHANGELOG.md>` is the same as simply `CHANGELOG.md`

Reply:
I kept that structure in case we were going to move to another location some repositories